community/planning/griddle/openapi.yaml

```diff
@@ -0,0 +1,2146 @@
+          schema:
+            type: string
+        - $ref: "#/components/parameters/service_id"
+      responses:
+        "200":
+          description: |
+            Successful request. The response will include a JSON object
+            representing the organization.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Organization"
+        "400":
+          $ref: "#/components/responses/400BadRequest"
+        "404":
+          $ref: "#/components/responses/404NotFound"
+        "500":
+          $ref: "#/components/responses/500ServerError"
+        "503":
+          $ref: "#/components/responses/503ServiceUnavailable"
+    put:
+      tags:
+        - Pike
+      summary: Updates a single organization with the given ID
+      operationId: put_organization
+      parameters:
+        - name: id
+          in: path
+          description: ID of the organization to fetch
+          required: true
+          schema:
+            type: string
+        - $ref: "#/components/parameters/service_id"
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/PostOrganizations"
+        description: A JSON list of Organizations
+        required: true
+      responses:
+        "202":
+          description: |
+            The organization was submitted for validation but has not yet been
+            evaluated. After the server receives the organization for validation, it
+            will attempt to execute the transaction contained in the batch.
+            If the transaction is successful, the distributed ledger's
+            state will be updated and Grid will receive a state delta event from the
+            distributed ledger. The Grid Daemon processes this event and updates
+            the Grid database accordingly.
+            This means that there is a delay between when transactions are
+            submitted and when the new data is actually available to fetch using
+            the Grid REST API GET endpoints. This delay can range from a few
+            milliseconds to several minutes depending on the underlying ledger.
+            Developers should take this delay into account when designing apps
+            on top of Grid.
+          content:
+            application/json:
+              schema:
+                properties:
+                  link:
+                    $ref: "#/components/schemas/BatchPostResponse"
+        "400":
+          $ref: "#/components/responses/400BadRequest"
+        "500":
+          $ref: "#/components/responses/500ServerError"
+        "503":
+          $ref: "#/components/responses/503ServiceUnavailable"
+  /organizations/{org_id}/agents:
+    get:
+      tags:
+        - Pike
+      summary: Lists all agents
+      operationId: list_agents
+      parameters:
+        - name: org_id
+          in: path
+          description: ID of the organization to fetch
```

Comment:
could maybe just remove "to fetch" or replace with "the agent belongs to"

Reply:
Would say the same for other instances of this file where it describes the organization as "to fetch" outside of the fetch org endpoint. Could maybe just replace with "ID of the org the X belongs to" which seems contextually more accurate